docs/resources.html

```diff
@@ -272,32 +285,46 @@ <h2 id="improving-the-quality-of-reviews">Improving the quality of reviews</h2>
+				
 				<h3 id="publishing-scientific-criticism">Publishing scientific criticism</h2>
 				<p>In this context, “scientific criticism” means a formal written response to a published finding. Journals can handle scientific criticism in different ways, e.g., publishing letters, commentaries, or online comments (post-publication peer review, PPPR). <a href="https://doi.org/10.31222/osf.io/4w3kb" target="_blank">Hardwicke et al. (2022)</a> assessed PPPR policies and practices at journals and found that most journals don’t currently offer options for submitting PPPR, and those that do tend to impose strict length/timing limits. They also found that even at journals that do offer PPPR, very few PPPRs actually end up being published. Hardwicke and colleagues make several recommendations for journals that are interested in adopting PPPR in <a href="https://doi.org/10.31222/osf.io/4w3kb" target="_blank" title="Hardwicke et al., 2022">Box 2 of their article</a>.</p>
+				
 				<h2 id="peer-review-innovations">Peer review innovations</h2>
 				<p>Peer review is undergoing a sustained and multifaceted debate about different attempts to improve its perceived effectiveness, objectivity, transparency, and long-term sustainability. Many of these have already been discussed extensively above, e.g. deanonymizing reviews and/or reviewers, acknowledging and incentivizing reviewing, etc. There are many more – for an analytical overview of ongoing peer review innovations see <a href="http://www.doi.org/10.31235/osf.io/8hdxu" target="_blank">Kaltenbrunner et al. (2022)</a>. They categorize peer review innovations as either being about the <em>object</em> of peer review (what is being peer reviewed), the <em>aim</em> of peer review (why is peer review being performed), the <em>role</em> of peer review <em>actors</em> (who performs peer review), the <em>nature</em> of peer review (how is peer review performed), or the <em>openness/transparency</em> of peer review (what is available to whom during and after peer review).</p>
+				<p>One such innovation is the peer review of preprints. ASAPbio has <a href="https://asapbio.org/recognizing-preprint-peer-review">some resources</a> on how we can improve recognition of preprint peer review.</p>
```

Comment:
Link text should be meaningful for accessibility (e.g., include ASAPbio has in the `<a>` tag. Alternatively, specify a link title as done in some cases above. Also, `target="_blank"`